pkg/connector/groups.go

```diff
@@ -20,16 +21,21 @@ type groupBuilder struct {
 }
 
 func groupResource(group *gitlabSDK.Group) (*v2.Resource, error) {
+	profile := map[string]interface{}{
+		"id":   group.ID,
+		"name": group.Name,
+	}
+	if group.ParentID != 0 {
+		profile["parent_group_id"] = group.ParentID
+	}
+
 	return resourceSdk.NewGroupResource(
 		group.Name,
 		groupResourceType,
-		group.ID,
+		strconv.Itoa(group.ID)+"/"+group.Name,
```

Comment:
Because the resource IDs for groups have changed in this PR, you'll need to update CI tests with the new resource/entitlement/grant IDs. It looks like the current ones just have the group ID, not the group name.